apps/recovery-portal/src/main/webapp/password-reset-complete.jsp

```diff
@@ -62,6 +62,7 @@
     String username = null;
     PreferenceRetrievalClient preferenceRetrievalClient = new PreferenceRetrievalClient();
     Boolean isAutoLoginEnable = preferenceRetrievalClient.checkAutoLoginAfterPasswordRecoveryEnabled(tenantDomain);
+    String sessionDataKey = null;
```

Comment:
shall we use `String sessionDataKey = StringUtils.EMPTY;`
or
`String sessionDataKey = "";` instead?